readme.md

```diff
@@ -119,33 +161,56 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
+Clark Answer: Methods with and ! attached to the end of them usually mean that they will
+modify the object they are calling on. Save! will raise an error if not successful. Save
+will return true or false
 
 ## Question 10
 
-Using jQuery, write an AJAX request to `http://tunr.com/artists` that would create a new artist with the name of 'Resin Laying Deer Figurine, Gold', and pop up a box saying "All done!" when complete.
+Using jQuery, write an AJAX request to `http://tunr.com/artists` that would create a new
+artist with the name of 'Resin Laying Deer Figurine, Gold', and pop up a box saying "All
+done!" when complete.
 
 ```js
-// Your code starts here...
+$.ajax({
+  url: 'http://tunr.com/artists',
+  method: 'post',
+  data: {name: 'Resin Laying Deer Figurine, Gold'},
+  contentType: 'application/json'
+}).done(function(res){
+  alert('All Done!')
+})
 
 ```
 
 ## Question 11
 
-Due to budget cuts, GA can no longer hire new instructors. Instead they can only instantiate new instructors with Javascript.
+Due to budget cuts, GA can no longer hire new instructors. Instead they can only instantiate
+new instructors with Javascript.
 
-Define a Javascript constructor called 'Instructor'. Every instance of Instructor should have a `name` property, and a method called `receivePresent`. This method takes one argument called `gift` and, when executed, console-logs "[name] promptly drops the [gift] on the floor."
+Define a Javascript constructor called 'Instructor'. Every instance of Instructor should
+have a `name` property, and a method called `receivePresent`. This method takes one argument
+called `gift` and, when executed, console-logs "[name] promptly drops the [gift] on the
+floor."
 
-Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
+Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin
+Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
+function Instructor(name, gift){
+   var self = this;
+   this.name = name;
+   this.receivePresent = function (gift){
+     console.log(self.name + "promptly drops the "+ gift +" on the floor.")
+   }
+ }
 
 ```
```

Comment:
Although it would be better to define `receivePresent` as a prototypal method so that we aren't taking up memory with multiple copies of the method.

```js
Instructor.prototype.receivePresent = function(gift){
  console.log(self.name + "promptly drops the "+ gift +" on the floor.")
}
```

The question also asks you to instantiate a new instructor. Take a look at the prompt again and see if you can do it!